presto-docs/src/main/sphinx/connector/teradata.rst

```diff
@@ -0,0 +1,53 @@
+===================
+Teradata  Connector
+===================
+
+Teradata provides a low latency high performing connector that
+supports high concurrency, and parallel processing between Teradata
+and Presto. The Teradata QueryGrid connector for Presto enables users
```

Comment:
Connector names are a bit confusing, I thought Querygrid connector is the one that is to connect from Teradata to Presto and Teradata connector is the reverse direction. Do we want to clarify the naming convention in the beginning?